.github/workflows/ci.yml

```diff
@@ -10,7 +10,9 @@ on:
 
 env:
   # Common versions
+  # renovate: datasource=repology depName=homebrew/go versioning=loose
```

Comment:
What does homebrew mean here?

This is probably a question best answered with a comment in the file explaining what these directives are doing and perhaps pointing to the renovate config file.

Reply:
Renovate can be configured to check the latest version available for a package from different sources, see [docs](https://docs.renovatebot.com/modules/datasource/). The most commonly used seems to be "repology" which keeps track of all the versions available from almost all the available sources, e.g. homebrew, this means that renovate will check the latest version available from homebrew for go and golangci-lint. I don't have strong opinions regarding the repository chosen, homebrew felt like a safe bet. I just noticed there is also a golang-version datasource we could use, but still we'd need to handle differently golangci-lint, either through repology or directly from the GitHub repo, so I'm not sure it would be worth switching. I agree a comment could be useful, I'll add that.

Reply:
Instead of adding a comment I moved the whole logic to the `renovate.json` file, comments in the workflows files felt too brittle. I also switched to `golang-version` and `github-tags` as datasources for `go` and `golangci-lint` versions respectively. Inspired by [this](https://github.com/timescale/promscale/blob/master/.github/renovate.json) and other examples found grepping through Github.